dspace/config/dspace.cfg

```diff
@@ -643,7 +643,7 @@ event.dispatcher.default.class = org.dspace.event.BasicDispatcher
 # Add doi here if you are using org.dspace.identifier.DOIIdentifierProvider to generate DOIs.
 # Adding doi here makes DSpace send metadata updates to your doi registration agency.
 # Add rdf here, if you are using dspace-rdf to export your repository content as RDF.
-event.dispatcher.default.consumers = versioning, discovery, eperson, harvester
+event.dispatcher.default.consumers = versioning, fileaccess, discovery, eperson, harvester
```

Comment:
Another minor note: Not sure this `fileaccess` consumer should be enabled by default if the rest of this feature is disabled by default